SamplifyAPIClient/Teams.cs

```diff
@@ -0,0 +1,19 @@
+using System.Runtime.Serialization;
+
+namespace Dynata.SamplifyAPIClient
+{
+    [DataContract]
+    public class Team
+    {
+        [DataMember(Name = "id")]
+        public int ID { get; set; }
+        [DataMember(Name = "name")]
+        public string Name { get; set; }
+        [DataMember(Name = "default")]
+        public bool Default { get; set; }
+        [DataMember(Name = "description")]
+        public string Description { get; set; }
+        [DataMember(Name = "status")]
+        public string Status { get; set; }
```

Comment:
Can this be defined with constants?

Reply:
This status here is a part of response. Could you help me understand how can I define a constant here? 
The value of Status here can be either `ACTIVE` or `INACTIVE`.